### ISSUE TITLE
Export StatsPlots.jl

### ISSUE THREAD

**Issue by dpsanders** [JuliaStats/StatsKit.jl#18]
It feels like this package should also re-export `StatsPlots.jl`.

**Comment by nalimilan** [comment]
That's going to make the load time even worse, but... :-) Could you make a PR?

**Comment by ararslan** [comment]
I don't think we should endorse a particular plotting ecosystem here; including StatPlots is effectively saying, "Plots is how you plot data!" when in fact many people (myself included) greatly prefer alternatives such as Gadfly or Winston.

**Comment by oxinabox** [comment]
I think we should  revisit this discussion. StatsPlots isn't that huge, and it can substantially improve quality of life.
For Plots.jl users who I think are the majority.

Further, there is no equivalent for Winston, or VegaLite,
Gladly does some but not most intrinistivally.
and Makie recipies will likely end up in StatsPlots anyway.

If we really wanted we could wrap it in a Requires. So it doesn't get loaded until Plots.jl is loaded, but then Plots.jl will just work better . Though that feels a bit magic.
